### ISSUE TITLE
Add documentation about how mined tuples are used by each loss function

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-metric-learning#202]
The docs should have the following type of explanation, plus a short note per loss function

 - If you pass pairs into a triplet loss, then triplets will be formed by combining each positive pair and negative pair that share the same anchor.
 - If you pass triplets into a pair loss, then pairs will be formed by splitting each triplet into two pairs
 - If you pass pairs or triplets into a classification loss, then each embedding's loss will be weighted by how frequently the embedding occurs in the pairs or triplets.
